src/pantry.js

```diff
@@ -4,20 +4,29 @@ class Pantry {
     // console.log(this.userIngredients);
   }
   createPantry(data) {
-    this.userIngredients = this.userIngredients.map((ingredient) => {
+    // this.userIngredients = this.userIngredients.map((ingredient) => {
```

Comment:
This can still be removed, but I am glad to see we are holding on to both ways of doing it for a minute at least.